prospero-common/src/main/java/org/wildfly/prospero/api/ProvisioningDefinition.java

```diff
@@ -69,7 +70,7 @@ private ProvisioningDefinition(Builder builder) throws ArtifactResolutionExcepti
                 this.includedPackages.addAll(featurePackInfo.packages);
                 this.repositories.addAll(featurePackInfo.repositories);
                 setUpBuildEnv(channelRepo, provisionConfigFile, channel, featurePackInfo.channelGav);
-            } else {
+            } else if (provisionConfigFile.isPresent()) {
```

Comment:
This is just resolving unchecked `provisionConfigFile.get()` on line 76 bellow, which could produce a NPE.